en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+# Managing Common API Policies
+
+## Add common API Policies in an environment
+
+You can add common API Policies via the Publisher Portal.
+However, **WSO2 API Controller (apictl)** allows you to create API Policies without using the Publisher Portal. For more information on adding common API Policies, see [Importing Common API Policies Via Dev First Approach]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/importing-common-api-policies-via-dev-first-approach).
+
+## Get Common API Policies in an environment
+
+Follow the instructions below to display a list of common API Policies in an environment using apictl:
+
+1.  Make sure that the WSO2 API Manager (WSO2 API-M) 4.1.0 version is started and that the 4.1.0 version of apictl is set up.   
+     For more information, see [Download and Initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
+2.  Log in to the WSO2 API-M in the environment by following the instructions in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
+3.  Run the corresponding apictl command below to get (list) the common API Policies in an environment.
+
+    1. Get API Common Policies in an environment.
+
+        -   **Command**
+            ``` bash
+            apictl get policies api -e <environment>
+            ```
+            ``` bash
+            apictl get policies api --environment <environment>
+            ```
+            ``` bash
+            apictl get policies api --environment <environment> --all 
+            ```
```

Comment:
Is it better to have tabs here for these code blocks? They all do the same thing with some minor differences that we should explain.

Reply:
Please add titles for each tab. Not sure if this will render, but here's an example of what I'm talking about:

Example: 
``` bash tab="Bash"
#!/bin/bash

echo "Hello world!"
```

``` c tab="C"
#include <stdio.h>

int main(void) {
  printf("Hello world!\n");
}
```

``` c++ tab="C++"
#include <iostream>
```

Reply:
Same comment as in https://github.com/wso2/docs-apim/pull/6103#discussion_r944140879